client/client.go

```diff
@@ -52,3 +52,11 @@ func NewClientWithConn(conn *grpc.ClientConn) *Client {
 		cli: cpb.NewContainerzClient(conn),
 	}
 }
+
+// NewClientFromGNOI allows the creation of a client using a client
```

Comment:
Maybe call this `NewWithStub` since it doesn't matter where you got the `cpb.ContainerzClient` from AFAICT?